packages/RichTextEditing-Core.package/RichTextDocument.class/instance/charcountExcept..st

```diff
@@ -0,0 +1,14 @@
+count
+charcountExcept: aListOfBytecodes
+	
+	| byteString count i|
+	count := 0.
+	byteString := self text string asByteArray.
+	
+	i := 1.
+	[i <= byteString size] whileTrue:[
+		(aListOfBytecodes includes: (byteString at: i)) ifFalse: [count := count+1].
```

Comment:
^ byteString count: [:each | (aListOfBytecodes includes: each) not]

Reply:
nice!